src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,35 @@
+package duke.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Task type event.
+ */
+public class Event extends Task {
+
+    private final LocalDateTime at;
+    private final DateTimeFormatter formatter =
+            DateTimeFormatter.ofPattern("MMM dd yyyy, HH:mm");
+
+    public Event(String description, LocalDateTime at) {
+        super(description);
+        this.at = at;
+    }
+
+    @Override
+    public String getSaveTime() {
```

Comment:
Shouldn't the header comments be included as this is a public method? Similar problems encountered for other methods in this class.

Reply:
This is a get method, comments omitted.